README.markdown

```diff
@@ -30,7 +30,7 @@ Note: On OS X, if you have/had a "system" node installed and want to install mod
 
 Homebrew installation is not supported. If you have issues with homebrew-installed `nvm`, please `brew uninstall` it, and install it using the instructions below, before filing an issue.
 
-Note: If you're using `zsh` you can easily install `nvm` as a zsh plugin. Install [`zsh-nvm`](https://github.com/lukechilds/zsh-nvm) and run `nvm_update` to update.
+Note: If you're using `zsh` you can easily install `nvm` as a zsh plugin. Install [`zsh-nvm`](https://github.com/lukechilds/zsh-nvm) and run `nvm upgrade` to upgrade.
```

Comment:
Ok, the only way I can think to do this is to save the output of `nvm upgrade` into a variable and compare it against the output of `nvm --help`. If they match, `nvm` doesn't have upgrade functionality yet so do it in `zsh-nvm`, if they don't match then `nvm` handled the update so print the output.

Once the update functionality is added to `nvm` I could hardcode a version to check against in `zsh-nvm` instead of comparing command outputs.

Does that sound reasonable?